README.md

```diff
@@ -1 +1,167 @@
-# To-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <br/>
+
+  <h3><b>TO-DO-LIST README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [Key Features](#key-features)
+- [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [TO_DO_LIST] <a name="about-project"></a>
+The goal of this project to use plain JavaScript with objects. In this project, we build a basic website that allows users to add/remove books from a list. We achieved that by using JavaScript objects, arrays, dynamically modify the DOM and add some basic events.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Webpack
+- HTML, CSS & JavaScript
+- Git, Github and Visual Studio Code
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[I implement HTML5 semantics and CSS3]**
+- **[Javascript, ES6.]**
+- **[Webpack and webpack dev server]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](N/A)
+
+## 🚀 Live Video <a name="live-video"></a>
+
+- [Live Video Link](N/A)
+
```

Comment:
As you have not deployed the project so it's showing a 404 error. If you want to deploy you can do it with GitHub pages or simply remove this section in those projects where you don't deploy the project online.

Reply:
Thank You